src/lib/registration/storage.rb

```diff
@@ -37,7 +37,7 @@ class RegCodes < Struct.new(:reg_codes)
 
       def initialize
         self.reg_codes = if Stage.initial
-          reg_codes_from_usb_stick || {}
+          reg_codes_from_usb_stick || reg_codes_from_install_inf || {}
```

Comment:
So you cannot use both features together... If it is OK then it should be at least documented somewhere.

Reply:
yes. By intention (we have to set rules for possible collisions somehow ...)